tomviz/MainWindow.cxx

```diff
@@ -990,4 +1001,25 @@ void MainWindow::syncPythonToApp()
   sync.call(args, kwargs);
 }
 
+void MainWindow::findPipelineTemplates() {
+  m_pipelineTemplates->clear();
+
+  QDir dir = QApplication::applicationDirPath() + "/../share/tomviz/templates/";
+  foreach (QString file, dir.entryList()) {
+    QString fileName = file.split(".")[0];
+    QString menuName = file.split(".")[0].replace("_", " ");
```

Comment:
If you use `entryInfoList` here this is all provided in `FileInfo` without the need to `split` anything.

Reply:
I seem to have overlooked that function completely at first, thank you. Fixed.